### PR TITLE
fix: remove expose

### PR DESCRIPTION
**Pull request by AAfghahi**
### SUMMARY

When creating a filter for the Database Filter view we thought that we would have to add individual filters into the search_filter column. But we learned that we don't. This is cleaning that up. 

### ADDITIONAL INFORMATION

- [ ] Has associated issue:
- [ ] Required feature flags:
- [ ] Changes UI
- [ ] Includes DB Migration (follow approval process in [SIP-59](https://github.com/apache/superset/issues/13351))
  - [ ] Migration is atomic, supports rollback & is backwards-compatible
  - [ ] Confirm DB migration upgrade and downgrade tested
  - [ ] Runtime estimates and downtime expectations provided
- [ ] Introduces new feature or API
- [ ] Removes existing feature or API

### CODE REVIEW

**Comment by jinghua-qa** [comment]
Tested in the ephemeral env, when i try to save the change to allow data upload in db setting , error msg will show and change can not be made.
Screen Shot 2022-04-15 at 11 24 16 AM

**Comment by jinghua-qa** [comment]
> Tested in the ephemeral env, when i try to save the change to allow data upload in db setting , error msg will show and change can not be made. Screen Shot 2022-04-15 at 11 24 16 AM

This is due to the sqlite in ephermal env, will test again locally

**Comment by jinghua-qa** [comment]
Tested PR locally and it worked as expected

https://user-images.githubusercontent.com/81597121/163622256-d7388e99-50e5-414f-b200-d7c821075f2f.mov

File: superset/databases/filters.py [hunk 1]
```diff
@@ -69,8 +69,6 @@ class DatabaseUploadEnabledFilter(BaseFilter):  # pylint: disable=too-few-public
     def apply(self, query: Query, value: Any) -> Query:
         filtered_query = query.filter(Database.allow_file_upload)
 
-        database_perms = security_manager.user_view_menu_names("database_access")
-        schema_access_databases = can_access_databases("schema_access")
```
Comment (eschutho) [inline superset/databases/filters.py:73]: if I compare this filter to the actual csv file upload validator, the current code checks for 
`security_manager.can_access_database(database)`
and 
schema in `database.get_schema_access_for_file_upload()`

**Review by eschutho — APPROVED** [review]

**Comment by sadpandajoe** [comment]
🏷️  preset:2022.15

**Pull request merged**